### PR TITLE
Fixup package install

### PR DESCRIPTION
**Pull request by pmatulis**

### CODE REVIEW

File: src/en/installconfig-package-install.md [hunk 1]
```diff
@@ -1,33 +1,38 @@
...
+Each of the above packages has its own dependencies. That is, each will bring
+in other MAAS packages not listed above. The full list of MAAS packages can be
+obtained with the command:
 
-If you want to distribute these services on several machines (or want to
-[deploy extra rack controllers](./installconfig-rack.html#add-a-rack-controller)),
-you will need to install packages individually on those machines.
+```bash
+apt-cache search maas-
+```
```
Comment (degville) [inline src/en/installconfig-package-install.md]: Not sure the '-' is needed (it misses out python3-django-maas, for example).

File: src/en/index.md [hunk 2]
```diff
@@ -83,33 +83,31 @@ was stated earlier still applies. For instance, if Juju removes a machine then
...
+rack controller. See [Concepts and terms](intro-concepts.html#controllers) for
+how each are defined.
 
-- Region controller(s)
-- Rack controller(s)
-- MAAS machines (nodes)
+Unless there is specific reason not to, it is recommended to have both
+controllers residing on the same system. A no-fuss way to achieve this is by
+installing the `maas` metapackage (or while installing Ubuntu Server with an
+ISO).
```
Comment (degville) [inline src/en/index.md]: The Ubuntu Server bit doesn't quite make sense to me, perhaps:
A no-fuss way to achieve this is by installing the `maas` metapackage, or by installing from the Ubuntu Server ISO.

**Comment by pmatulis** [comment]
@degville Thanks for the comments. I fixed.

**Comment by degville** [comment]
Looks good. A great update Peter - thanks! :cake:

**Pull request merged**